<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslatorWindow</class>
 <widget class="QMainWindow" name="TranslatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translator App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QTextEdit" name="from_text"/>
     </item>
     <item>
      <widget class="QPushButton" name="paste_btn">
       <property name="font">
        <font>
         <family>Carlito</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(98, 114, 164);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Paste</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Carlito</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(68, 71, 90);</string>
       </property>
       <property name="text">
        <string>By Ermias Bahru</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Carlito</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(68, 71, 90);</string>
       </property>
       <property name="text">
        <string>V 1.0.1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QTextEdit" name="to_text"/>
     </item>
     <item>
      <widget class="QPushButton" name="copy_btn">
       <property name="font">
        <font>
         <family>Carlito</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(98, 114, 164);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="translate_btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(189, 147, 249);
	border-radius: 6px;
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgba(189, 147, 249, 230);
}</string>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(189, 147, 249);
	border-radius: 6px;
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgba(189, 147, 249, 230);
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>To</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
